about.html

```diff
@@ -0,0 +1,147 @@
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
+        </div>
+        </div>
+
+         <!-- hamburger menu contents -->
+
+        <nav class="mobile-div" id="mobile">
+            <ul id="Mobile-menu">
+              <li id="close-menu"><img src="./assets/vectorx.png" alt="close button"></li>
+              <li> <a class="menuItems" id="menu1" href="./index.html">Home</a></li>
+              <hr>
+              <li> <a class="menuItems" id="menu2" href="#our-goals">Our Goals</a></li>
+              <hr>
+              <li> <a class="menuItems" id="menu3" href="#featured-speakers">Guests</a></li>
+              <hr>
+              <li> <a class="menuItems" id="menu3" href="#partners">Partners</a></li>
+              <hr>
+            </ul>
+        </nav>
+
+        <!-- hamburger menu Icon and header page --> 
+         <div>
+            <nav id="desktop-nav-bar">
+                <ul class="desktop-items">
+                 <li class="top-logo"> <a class="nav" href="#home"><div class="loggo-div">
+                    <img class="topl-img" src="./assets/1.png" alt="Logo">
+                    <h5 class="toplh">SUPER FOOD
+                        <P class="topltxt">Be Smart, Eat Smart</P>
+                    </h5>
+                </div>
+            </a></li>
+                  <li class="list-nav"> <a class="nav navl" href="./index.html">Home</a></li>
+                  <li class="list-nav"> <a class="nav navl" href="#home">Program</a></li>
+                  <li class="list-nav"> <a class="nav navl" href="#contactus">Join</a></li>
+                  <li class="list-nav"> <a class="nav navl" href="#">Sponsor</a></li>
+                  <li class="list-nav"> <a class="nav navl" href="#Events">News</a></li>
+                  <li class="list-nav nav"><button class="sf-btn">SF Campaign</button></li>
+                </ul>
+            </nav>
+            </div>
+
+        <div class="menuBar"> <i class="fa fa-bars"></i></div>
+    </header>
+    <section>
+        <div class="heading-text" id="home">
+            <p class="head-para mt-4 my-0 h5">"Hello! Sharing World"</p>
+        </div>
+        <h1 class="mt-2">SUPER FOOD</h1>
+        <div class="main-para-div">
+        <p class="main-para">We will provide a well balanced food and tell you why you need to eat that, through Robo-Vending machines, food delivery, hotels and mobile vending machines. We will make people aware of what they are eating now and why to choose us through social media marketing, online marketing, etc.
+        </p>
+       </div>
+        <div class="mainp-div" id="contactus">
+        <p class="main-para2">Please contact our E-mail for further questions about Super Food.
+            <a href="#">superfoods@foods.org</a>
+        </p>
+    </div>
+    </section>
```

Comment:
- [ ] Hi there you did a really good Job. As part of the requirements, you are supposed to follow the design as much as possible so, in the top section of the About page of the design there's a background image but its not present in your about page please kindly consider adding one

design 
<img width="965" alt="image" src="https://user-images.githubusercontent.com/37901469/234852770-fd2375c0-5d65-456e-9f46-f5415ac89442.png">


Your App
<img width="965" alt="image" src="https://user-images.githubusercontent.com/37901469/234852865-8c903240-d95a-4992-8f22-1c2dc71dc226.png">